packages/design-system/src/components/ThirdPartyExternalLink/ThirdPartyExternalLink.tsx

```diff
@@ -66,8 +66,4 @@ const ThirdPartyExternalLink = ({
   );
 };
 
-ThirdPartyExternalLink.defaultProps = {
-  learnMoreUrl: '',
```

Comment:
i don't know why this was an empty string but it seems like if we were to define a default value, it should be useful